src/App.tsx

```diff
@@ -47,24 +50,26 @@ const JUP_MAIN = "#009046";
 const BODY_DARK = "#0a1c13";
 
 // Login page wraps a private route for everything else. The private route determines if a user is logged in.
+// NOTE: snackbarprovider needs to be the highest parent to work properly
+// TODO: Is it possible to lower snackbarprovider in the app's heirarchy for less renders?
 const App: React.FC = () => {
   return (
-    <AuthProvider>
-      <Router>
-        <MUIThemeProvider>
-          <Routes>
-            <Route path="/" element={<Login />} />
-            <Route path="/dashboard" element={<Private Component={Dashboard} />} />
-            <Route path="/transactions" element={<Private Component={Transactions} />} />
-            <Route path="/portfolio" element={<Private Component={Portfolio} />} />
-            <Route path="/peers" element={<Private Component={Peers} />} />
-            <Route path="/exchange" element={<Private Component={DEX} />} />
-            <Route path="/blocks" element={<Private Component={Blocks} />} />
-            <Route path="/generators" element={<Private Component={Generators} />} />
-          </Routes>
-        </MUIThemeProvider>
-      </Router>
-    </AuthProvider>
+    <SnackbarProvider maxSnack={MaximumSnackbarMessages}>
```

Comment:
I don't have a problem with this being here. the reason i had raised it to the index in the notistack changes was so that we couldn't mess up it's hierarchy and put it below something that needed to call it. fyi of course, your solution  here is exactly the same.

Reply:
Oh this was a mistake on my part then, I thought I was going with the way you had it. oops!